src/AudioUnit/AudioUnit.cs

```diff
@@ -1802,7 +1804,9 @@ public enum AURenderEventType : byte
 	[iOS (9,0), Mac (10,11)]
 	public enum AudioComponentInstantiationOptions : uint {
 		OutOfProcess = 1,
-		InProcess = 2
+		InProcess = 2,
+		[iOS (14,5), Mac (11,3)]
+		LoadedRemotely = 1u << 31,
```

Comment:
it's added in the tvOS assembly, should it be ?
if so there should be an availability attribute or a `[NoTV]`